quint/cli-tests.md

```diff
@@ -211,3 +211,16 @@ Temporarily disabled.
 
 <!-- !test check SuperSpec - Types & Effects-->
     quint typecheck testFixture/SuperSpec.qnt
+
+### OK REPL tutorial
+
+The REPL tutorial is reproducible in REPL.
+
+<!-- !test check REPL tutorial -->
+```
+if [ "$UNAME" == "Linux" -o "$UNAME" == "Darwin" ]; then
+  quint -q -r ../tutorials/repl/kettle.qnt::kettle <../tutorials/repl/replTestIn.txt \
+    | diff - ../tutorials/repl/replTestOut.txt
+fi
+# else diff does not work as expected on windows
+```
```

Comment:
I'd rather move this to `io-cli-tests.md` which are not run on windows anyway. Then we can get rid of the `if` statement.

Reply:
oh, I knew that one of them should be win-free :)